### PR TITLE
Update ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel

### PR DESCRIPTION
**Pull request by bwaldon**
~Will require merge conflict resolution before merge~

### CODE REVIEW

**Comment by bwaldon** [comment]
todo: remove stale pdf and tex

File: datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel [hunk 1]
```diff
@@ -1,4 +1,76 @@
...
+                :Head (D :t "the"))
+            :Head (Nom
+                :Head (N :t "extent")
+                :Comp (Clause
+                    :Marker (Sdr :t "that" :note "treating 'to the extent that' as an idiom w/ clausal complement")
+                    :Head (Clause
+                        :Subj (NP
+                            :Head (Nom
+                                :Head (N :t "funds" :l "fund")
+                                :Mod (PP
```
Comment (nschneid) [inline datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel:48]: not 100% sure about Mod vs. Comp here

File: datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
+                                        :Mod (DP
+                                            :Head (D :t "such"))
+                                        :Head (Nom
+                                            :Head (N :t "expense"))))))
+                        :Head (VP
+                            :Head (V_aux :t "are" :l "be" :xpos "VBP")
+                            :Comp (Clause
+                                :Head (VP
+                                    :Head (V :t "received" :l "receive" :xpos "VBN")
+                                    :Comp (PP
```
Comment (nschneid) [inline datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel:60]: why Comp?

File: datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel (hunk 1 above)
Comment (bwaldon) [reply datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel:48]: Thinking about this more, this seems more `:Comp` -- updated in https://github.com/nert-nlp/legal-cgel/pull/82/commits/4187cc47fa7e48c9086d152a18aefa4bb3cae703

File: datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel (hunk 2 above)
Comment (bwaldon) [reply datasets/oneoff/usc-042.ideef94fa8-2a49-11ef-8437-855ef5b8fc19_2_0.cgel]: Oh yeah, this should definitely be `:Mod` -- https://github.com/nert-nlp/legal-cgel/pull/82/commits/4187cc47fa7e48c9086d152a18aefa4bb3cae703

**Pull request merged**